src/hooks/core/chat/useFetchPostNotificationHook.ts

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
+import {Platform} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Platform` from 'react-native' is a good addition as it allows for platform-specific logic, which seems to be the main change in this pull request.
```diff
+ import {Platform} from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->